libskale/SnapshotManager.cpp

```diff
@@ -272,14 +272,36 @@ void SnapshotManager::leaveNLastSnapshots( unsigned n ) {
     }      // for
 }
 
+std::pair< int, int > SnapshotManager::getLatestSnasphot() const {
```

Comment:
why don't you just use directory name (it is block number) instead of time stamp?